cli/popper/log.py

```diff
@@ -100,7 +126,21 @@ def info(self, msg='', *args, **kwargs):
         """
         Logs a message with severity 'INFO'
         """
-        super(PopperLogger, self).info(msg, *args, **kwargs)
+        m = ""
+
+        if self.Color:
+            if (msg.find('[') != -1):
+                m = msg[0: msg.find("[") +1]
+                m += BOLD_CYAN
+                m += msg[msg.find("[")+1: msg.find("]")]
+                m += RESET
+                m += msg[msg.find("]"): ]
+            else:
+                m = msg
+        else:
+            m = msg
+
```

Comment:
Please check if so much of `else` statements can be avoided.

Reply:
done